### ISSUE TITLE
💡 Create CLI/NPX Tutorial Quest tool 

### ISSUE THREAD

**Issue by JacobMGEvans** [OpenSourceRaidGuild/tutorial-quest#41]
### Background & Motivation

I published the Tutorial Site to test a new tool, and was inspired to create a CLI tool for assisting in local setups for Raiding (something that can be useful in the future even after the Website platform is created) 

### Proposed Solution

N/A

### Detailed Design

Allows automate some setup needed for local Raiding.
- Should provide information on how to Raid
- Should provide Links to Tutorial Quest website, Discord Server (Raid Comms) 
- Should assist in Forking active Raid repo (how automated can this be?) 

### Help make it happen!

- [ ] I am willing to submit a PR to implement this change.
- [ ] I am willing to submit a PR to implement this change, but would need some guidance.
- [ ] I am not willing to submit a PR to implement this change.

**Comment by tigerabrodi** [comment]
@JacobMGEvans Will this be like a tool everyone can use in their terminal?

- Fork our Raid repo.
- Provide information on how to raid.
- Providing the links

All via the terminal, and providing would be the output in the terminal?

**Comment by JacobMGEvans** [comment]
> @JacobMGEvans Will this be like a tool everyone can use in their terminal?
> 
> - Fork our Raid repo.
> - Provide information on how to raid.
> - Providing the links
> 
> All via the terminal, and providing would be the output in the terminal?

Yes! That is the main idea. A tool for assisting in Raiding locally. Information on how to raid but also scheduled events like you mentioned maybe even a command for connecting to the Discord raid stream

**Comment by tigerabrodi** [comment]
@JacobMGEvans Sounds neat! :fire: 

You want to work on this together :laughing: 

Damn, a good moment to learn Rust lol

or do we plan on developing it using Node or Deno? :thinking:

**Comment by JacobMGEvans** [comment]
Lol, maybe the overhead would add to the time and maintenance. Overall we can figure that part out later.

We can work on it together.

**Comment by tigerabrodi** [comment]
@JacobMGEvans A'ight brother :orange_heart: :orange_heart: :fire:
